man/syncthing-faq.7

```diff
@@ -128,12 +128,6 @@ firewall\-setup to enable direct connections.
 Second, if one of the devices is a very low powered machine (a Raspberry Pi,
 or a phone, or a NAS, or similar) you are likely constrained by the CPU on
 that device. See the next question for reasons Syncthing likes a faster CPU.
-You can verify this by looking at the CPU utilization in the GUI. If it is
```

Comment:
The man pages are auto generated from the syncthing/docs repo, so the change should happen there instead.